prometheus-metrics-exposition-textformats/src/test/java/io/prometheus/metrics/expositionformats/ExpositionFormatsTest.java

```diff
@@ -2737,11 +2766,86 @@ public void testLabelValueEscape() throws IOException {
     assertPrometheusText(prometheus, counter);
   }
 
+  @Test
+  public void testFindWriter() {
+    ExpositionFormats expositionFormats = ExpositionFormats.init();
+
+    // delimited format - should use default escaping scheme
```

Comment:
good candidate for parameterized test - e.g. like here: https://github.com/prometheus/client_java/blob/3133f66c35dbe9dfa4bdfbc661a4b6580851521b/integration-tests/it-exporter/it-exporter-test/src/test/java/io/prometheus/metrics/it/exporter/test/ExporterIT.java#L92